packages/server/tests/acceptance/ws/subscribe.spec.ts

```diff
@@ -21,11 +21,11 @@
 // external resources
 import { solidity } from "ethereum-waffle";
 import chai, {assert, expect} from "chai";
+import WebSocket from 'ws';
 chai.use(solidity);
 
 import {Utils} from '../../helpers/utils';
-import { finished } from "stream";
-import bodyParser from "koa-bodyparser";
+import {predefined} from "@hashgraph/json-rpc-relay";
```

Comment:
I thought we didn't want to import from the package anymore to aid easy local testing

Reply:
Yes you are right!! 😅 
I will change it 👍